platform-core-module/src/main/java/com/softicar/platform/core/module/uuid/AGUuid.java

```diff
@@ -29,7 +32,17 @@ public IDisplayString toDisplayWithoutId() {
 		return EmfSourceCodeReferencePoints//
 			.getReferencePoint(getUuid())
 			.map(IEmfSourceCodeReferencePoint::toDisplay)
-			.orElse(IDisplayString.create(getUuidString()));
+			.orElse(getStaticRoleTitle(getUuid()));
+	}
+
+	// FIXME Should be replaced when roles can be created at runtime
+	private static IDisplayString getStaticRoleTitle(UUID uuid) {
```

Comment:
method doesnt need to be static
Also, should be moved to bottom below `private static` methods

Reply:
ok and ok